okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -387,11 +385,11 @@ public void run() {
           }
           clientFrameHandler = new ClientFrameHandler(testFrameReader);
           executor.execute(clientFrameHandler);
+          frameWriter.becomeConnected(testFrameWriter, socket);
```

Comment:
It should be called before startPendingStreams() below which will trigger sending.